servers/plugins/org.fusesource.ide.server.karaf.core/plugin.xml

```diff
@@ -346,32 +414,62 @@
        			<facet id="jst.web"  version="2.2,2.3,2.4,2.5,3.0"/>
        			<facet id="jst.java" version="1.3,1.4,5.0,6.0,1.7,1.8"/>
        			<facet id="jst.utility" version="1.0"/>
-       			<facet id="jst.camel" version="2.15"/>
```

Comment:
don't we have a bug here in current version of the product?

Reply:
yes, pretty much.

Reply:
What do you think to report it and create a PR separately?
As we don't know exactly when this PR will be merged (as amount of work to do on it is not well defined as it is "test and discover"), it will make this PR simpler and ensure that the fix is available and tested sooner.

Reply:
addressed in #1089